MedallionShell.NetCore/project.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+  "title": "MedallionShell",
+  "version": "1.0.0-*",
```

Comment:
@kal why is this version specified here? Does it matter that it doesn't match the version from AssemblyInfo?